### ISSUE TITLE
Bug:(platform) Link component - 1 link example is missing aria-label and title attributes

### ISSUE THREAD

**Issue by Seamoo13** [SAP/fundamental-ngx#3633]
#### Is this a bug, enhancement, or feature request?
bug
#### Briefly describe your proposal.

The link example for specifying relations between documents is missing the aria-label and title attributes. aria-label is needed for screen readers and title is needed for native tooltips. 

#### If this is a bug, please provide steps for reproducing it.
1. go to: https://fundamental-ngx.netlify.app/#/platform/link
2. look at the block for standard link examples
3. scroll down to the example for specifying relations between documents 
4. hover mouse over link (no tooltip) 
5. inspect element and look at attributes

![2020-10-15_1](https://user-images.githubusercontent.com/47522152/96148633-1f5d4900-0f11-11eb-8d8e-3dc47b138963.png)
